libavformat/riffdec.c

```diff
@@ -190,6 +193,9 @@ int ff_get_wav_header(AVFormatContext *s, AVIOContext *pb,
         channels         = 0;
         par->sample_rate = 0;
     }
+    /* override block_align for non mono ADPCM AICA */
```

Comment:
AICA in lavc use planar sample format, and thus there are more nibbles per single byte per channel. All this little details are important.